.github/workflows/make-demo-repo.yml

```diff
@@ -20,17 +20,14 @@ jobs:
       uses: actions/checkout@v2
       with:
         fetch-depth: 0  # avoids shallow checkout as needed by setuptools-scm
-    - name: Set up Conda environment
-      uses: conda-incubator/setup-miniconda@v2
+    - name: Set up Python
+      uses: actions/setup-python@v1
       with:
-        miniconda-version: "latest"
-        activate-environment: "pyscaffold"
-        environment-file: environment.yml
-        auto-update-conda: true
+        python-version: 3.7
         auto-activate-base: false
     - name: Build and create demo
       run: |
-        pip install -e . # otherwise templates are not packaged
+        python -m pip install -U setuptools pip -e .
```

Comment:
Didn't know this is possible in one line without a semicolon. So the update and installation of `.` is done in one step? Cool.

Reply:
I tested in the terminal a few times because I wanted the action to run as quick as possible (so less invocations of `pip`). It should work at least with a relatively recent version of `pip`.